server/src/test/java/org/apache/druid/server/NodeTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.server;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+
+public class NodeTest
```

Comment:
Would also be good to have an equals verification test here. Like this:

```java
  @Test
  public void testEqualsAndHashCode()
  {
    EqualsVerifier.forClass(Node.class)
        .usingGetClass()
        .verify();
  }
```

Reply:
And, the class is now just a facade on `DruidNode`, so we defer to the `equals()` and `hashCode()` methods on that class. (Though, we never actually use this class as a key: only a value.)